dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterable.java

```diff
@@ -150,18 +134,28 @@ public static ZonedIntervalIterable forConnectorArguments(
           "At least one day of query logs should be exported; you specified: " + daysToExport);
 
     LOG.info(
-        "Log entries within the last {} days will be exported in increments of 1 {}.",
+        "Log entries within the last {} days will be exported in increments of {} seconds.",
         daysToExport,
-        timeUnit);
+        DurationFormatUtils.formatDuration(duration.toMillis(), "**H:mm:ss**", true));
```

Comment:
Optional: I would suggest to use `DurationFormatUtils.formatDurationWords`. Also in other places.